applications/admin.py

```diff
@@ -181,7 +181,7 @@ class VideoInterviewSubmissionAdmin(SubmissionTypeAdmin):
     model = models.VideoInterviewSubmission
 
     def get_list_display(self, request):
-        return tuple(super().get_list_display(request) or ()) + ("video_file",)
+        return tuple(super().get_list_display(request) or ())
```

Comment:
I think we would want to include at least the interview id in the list display here. Even better would be including a link to the interview record in admin